_categories/sensors.md

```diff
@@ -7,15 +7,15 @@ desc: "How to add already implemented sensors (camera, lasers, IMU, ...) to the
 subcategories: 
   - title: "Altimeter Gazebo sensor"
     items: 
-      - title: 'Basic shapes in Gazebo simulator'
-        url: 'http://packages.osrfoundation.org/gz_big_index/4_sensors.html' # check
-        desc: 'How to insert and use an altimiter simulated sensor in Gazebo'
-        star: true
+      # - title: 'Basic shapes in Gazebo simulator'
+      #   url: 'http://packages.osrfoundation.org/gz_big_index/4_sensors.html' # check
+      #   desc: 'How to insert and use an altimiter simulated sensor in Gazebo'
+      #   star: true
 
-      - title: 'Camera Gazebo sensor'
-        url: 'http://gazebosim.org/tutorials?TODO' # check
-        desc: 'How to insert and use a simulated camera in Gazebo'
-        star: true
+      # - title: 'Camera Gazebo sensor'
+      #   url: 'http://gazebosim.org/tutorials?TODO' # check
```

Comment:
Found this - http://gazebosim.org/tutorials?tut=camera_save&cat=sensors#Createaworldwithacamera

This seems like the right url for this item.

Reply:
correct!